data/wanderer/wanderers start.txt

```diff
@@ -2867,6 +2867,6 @@ mission "Wanderers Ap'arak 3"
 	on complete
 		event "wanderers: the eye opens"
 		event "wanderers: derecho mass production" 16
-		event "wanderers: spera anatrusk colony" 40
+		event "wanderers: spera anatrusk colony" 90
```

Comment:
This change relies on the player completing the outfitter mission as fast as possible. Instead of wedging these mission within the delay between "Wanderers Ap'arak 3" and "Wanderers: Surveying 1," this delay should be created by the outfitter missions.